revpi-hat-eep/src/bin/revpi-eep.rs

```diff
@@ -167,15 +131,21 @@ fn create_rpi_eep(
 #[derive(Parser)]
 #[clap(author, version, about, long_about = None)]
 pub struct Cli {
-    /// The serial number for the device.
+    /// The serial number for the device. It is mandatory if the serial is not included in the
+    /// config file. This option will override the serial from the config file.
     #[clap(long, parse(try_from_str = parse_prefixed_int))]
-    pub serial: u32,
-    /// The end test date for the device. In the format YYYY-MM-DD (ISO8601/RFC3339). If omitted the current date is used.
-    #[clap(long, parse(try_from_str = parse_date_iso8601))]
-    pub edate: Option<chrono::NaiveDate>,
-    /// The (first) mac address of the device.
+    pub serial: Option<u32>,
+    /// The end test date for the device. In the format YYYY-MM-DD (ISO8601/RFC3339). If omitted the
+    /// current date is used. This option will override a given edate attribute from the config file.
     #[clap(long)]
-    pub mac: MacAddress,
+    pub edate: Option<NaiveDate>,
+    /// The (first) mac address of the device. It is mandatory if the mac is not included in the
+    /// config file. This option will override the mac from the config file.
+    #[clap(long)]
+    pub mac: Option<MacAddress>,
+    /// Full json configuration backup file name. The full json configuration includes also the serial, edate ann mac.
+    #[clap(long, value_parser, value_name = "BACKUP_CONFIG")]
+    pub backup: Option<PathBuf>,
```

Comment:
Suggestion: Would prefer to call it export_config_to_json or something like that. Backup feels like the wrong wording for this, even if it can be used as some sort of backup. Export feels more natural, as it also implies more, that someone could add it to a revision control system to track changes while backup is a save to disk and forget until the original file isn't available any more.

Later on both wordings are used/mixed, backup for the filename/path-string, backup_file for the variable holding the file pointer and the function export_config, which uses backup_file.